src/cursor/mod.rs

```diff
@@ -456,17 +439,20 @@ impl CursorData {
     }
 
     #[inline]
-    fn get_chunk_and_pos<'a>(&self, list: &'a BufList) -> Option<(&'a Bytes, usize)> {
+    fn get_chunk_and_pos<'a, 'b>(&'a self, list: &'b BufList) -> Option<(&'b Bytes, usize)> {
```

Comment:
I don't think this change is needed -- the old code should work.

Reply:
clippy agrees, fixed in subsequent commit